recycleIT-frontend/src/Components/Services/Map/Map.css

```diff
@@ -1,4 +1,20 @@
-  .Googl_Map {
-    margin-left: 450px;
+.map-side {
+  width: calc(100% - 380px);
+  height: 500px;
```

Comment:
Change that value to 100vh so that map is displayed correctly on desktop

Reply:
fixed the height of the map